### PR TITLE
3 qz xing building instruction and version information

### PR DESCRIPTION
**Pull request by dmykuliak-cat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+After that your QZXing.pro must look as:
+![QZXing.pro after editing](https://github.com/EDDragonWolf/QZXingBinaries/tree/master/images/QZXing.pro_after_editing.png).
+
+After build you can find your static binaries in shadow build directory:
+
+![Static binaries after build](https://github.com/EDDragonWolf/QZXingBinaries/tree/master/images/static_binaries_after_build.png)
+
+Go to the your project and create directory `staticBinaries`, and copied static binaries in that directory, open your project and link your static binaries:
+
+`QT += multimedia
```
Comment (EDDragonWolf) [inline README.md]: use ` ```<>``` `

**Review by EDDragonWolf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dmykuliak-cat) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,2 +1,72 @@
...
+
+CONFIG += c++11 qzxing_multimedia
+
+CONFIG(debug, debug|release) {
+    CONFIG+=qml_debug
+} else {
+
+...
+
+# Additional import path used to resolve QML modules in Qt Creator's code model
```
Comment (EDDragonWolf) [inline README.md]: Remove extra text

File: README.md [hunk 3]
```diff
@@ -1,2 +1,72 @@
...
+QML_IMPORT_PATH =
+
+#include(../../src/QZXing.pri)
+
+# Default rules for deployment.
+include(deployment.pri)
+
+...
+```
```
Comment (EDDragonWolf) [inline README.md]: Add statement for cloning repo

File: README.md [hunk 4]
```diff
@@ -1,2 +1,72 @@
...
+    TARGET.UID3 = 0xE618743C
+    TARGET.EPOCALLOWDLLDATA = 1
+    addFiles.sources = QZXing.dll
+    addFiles.path = !:/sys/bin
+    DEPLOYMENT += addFiles
+...
+```
+
+After build you can find your static binaries in shadow build directory.
+### For windows you'll get:
```
Comment (EDDragonWolf) [inline README.md]: Use bold, not header

**Review by EDDragonWolf — APPROVED** [review]

**Pull request merged**